packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/android/softwarerequirements.md

```diff
@@ -0,0 +1,15 @@
+* [Android Studio](https://developer.android.com/studio) 4.2.x or later
+* [JDK 8](https://www.oracle.com/java/technologies/javase/javase-jdk8-downloads.html) or later
+* [Apache Maven](https://maven.apache.org/download.cgi) 3.6.x or later
+
+    Include the following Okta Identity Engine Java SDK dependency for Apache Maven:
+
+    ```xml
```

Comment:
We should replace this with a gradle dependency block.
`implementation 'com.okta.idx.sdk: okta-idx-java-api:$okta_sdk_version'`